quickwit/quickwit-opentelemetry/src/otlp/mod.rs

```diff
@@ -131,6 +133,62 @@ fn is_zero(count: &u32) -> bool {
     *count == 0
 }
 
+pub fn extract_otel_traces_index_patterns_from_metadata(
```

Comment:
There is no need to convert the header value to a `String` since we subsequently split on `,`. We can work directly on `&str`.